rust_src/src/symbols.rs

```diff
@@ -179,6 +183,32 @@ impl LispSymbolRef {
     pub fn iter(self) -> LispSymbolIter {
         LispSymbolIter { current: self }
     }
+
+    // Set up SYMBOL to refer to its global binding.  This makes it safe
+    // to alter the status of other bindings.  BEWARE: this may be called
+    // during the mark phase of GC, where we assume that Lisp_Object slots
+    // of BLV are marked after this function has changed them.
```

Comment:
Could you make this a doc comment? i.e. make it use three slashes

Reply:
Done